tests/lib/helpers.js

```diff
@@ -10,7 +10,6 @@ var config = require('../lib/config');
 function done() {
     casper.options.pageSettings.userAgent = config.userAgent.phantomjs;
     casper.options.viewportSize = config.viewport.desktop;
-    casper.options.waitTimeout = 5000;
```

Comment:
Wait, this should probably remain and be changed to `casper.options.waitTimeout = 5000;`, for the case where we may set a test specific value?